hack/test-cmd.sh

```diff
@@ -48,10 +48,9 @@ trap "cleanup" EXIT
 set -e
 
 function find_tests {
-  cd "${OS_ROOT}"
-  find "${1}" -name '*.sh' | sort -u
+  find "${OS_ROOT}/test/cmd" -name '*.sh' | grep -E "${1}" | sort -u
 }
-tests=( $(find_tests ${1:-test/cmd}) )
+tests=( $(find_tests ${@:-sh}) )
```

Comment:
still only use the first arg here. I'd also expect `.*` as the default more than `sh`